README.md

```diff
@@ -28,6 +28,7 @@ variables:
 
 - HONEYCOMB_API_KEY - Your honeycomb API key
 - HONEYCOMB_DATASET - The name of the dataset you want to write to
+- $OTEL_EXPORTER_OTLP_ENDPOINT=`https://api.honeycomb.io`
```

Comment:
You can set `SERVICE_NAME` env var to set service name.

Reply:
would you like us to add that too? we didn't because each service has its own name